p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -655,6 +666,7 @@ mod tests {
     fn test_config<C: Scheme>(crypto: C, bootstrappers: Vec<Bootstrapper>) -> Config<C> {
         Config {
             crypto,
+            namespace: b"CW_P2P_AUTHENTICATED_TEST".to_vec(),
```

Comment:
nit: I recommend aligning this with the test config value: `_COMMONWARE_P2P_TEST_` (and optionally updating both to be clearer)